charts/mothership/values.yaml

```diff
@@ -65,7 +65,7 @@ bundler:
   enabled: false
   image: stackupwallet/stackup-bundler:v0.6.47
   loadBalancerExternal: false
-  gethRpc: http://node-1:8545
+  gethRpc: http://proxy:8080
```

Comment:
```suggestion
  gethRpc: http://proxyd:8080
```

Reply:
you're right. it should be `proxyd`

Reply:
Using `proxyd` to `blockscout` is currently being tested and may be applied in the future.